src/main/kotlin/hr/foi/pop/backend/controllers/AuthenticationController.kt

```diff
@@ -0,0 +1,119 @@
+package hr.foi.pop.backend.controllers
+
+import hr.foi.pop.backend.definitions.ApplicationErrorType
+import hr.foi.pop.backend.exceptions.*
+import hr.foi.pop.backend.models.user.User
+import hr.foi.pop.backend.models.user.UserLoginResponseDTO
+import hr.foi.pop.backend.models.user.UserMapper
+import hr.foi.pop.backend.request_bodies.LoginRequestBody
+import hr.foi.pop.backend.request_bodies.RefreshTokenRequestBody
+import hr.foi.pop.backend.request_bodies.RegisterRequestBody
+import hr.foi.pop.backend.responses.ErrorResponse
+import hr.foi.pop.backend.responses.SuccessResponse
+import hr.foi.pop.backend.responses.WarningResponse
+import hr.foi.pop.backend.security.jwt.TokenPair
+import hr.foi.pop.backend.services.AuthenticationService
+import hr.foi.pop.backend.services.UserService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/v2/auth")
+class AuthenticationController {
+    @Autowired
+    lateinit var userService: UserService
+
+    @Autowired
+    lateinit var authenticationService: AuthenticationService
+
+    @PostMapping("register")
+    fun registerUser(@RequestBody request: RegisterRequestBody): ResponseEntity<SuccessResponse> {
+        val savedUser = userService.registerUser(request)
+        val userDTO = UserMapper().mapDto(savedUser)
+
+        return ResponseEntity.status(HttpStatus.CREATED).body(
+            SuccessResponse("User \"${savedUser.username}\" registered with ID ${savedUser.id}.", userDTO)
+        )
+    }
+
+    @PostMapping("login")
+    fun loginUser(@RequestBody request: LoginRequestBody): ResponseEntity<out SuccessResponse> {
```

Comment:
When I try to login JPA will target table without specified schema which will result in exception:
![image](https://github.com/foivz/pop-app-backend/assets/61520141/47af4118-65f3-4153-bcf7-8c63baf84492)